modules/product/src/Event/AvailableVariationsEvent.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\commerce_product\Event\AvailableVariationsEvent.
+ */
+
+namespace Drupal\commerce_product\Event;
+
+use Drupal\commerce_product\ProductInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+class AvailableVariationsEvent extends Event {
+
+  /**
+   * The parent product.
+   *
+   * @var \Drupal\commerce_product\ProductInterface
+   */
+  protected $product;
+
+  /**
+   * An array of enabled variations.
+   *
+   * @var array
+   */
+  protected $variations;
+
+  /**
+   * Constructs a new AvailableVariationsEvent object.
+   *
+   * @param \Drupal\commerce_product\ProductInterface
```

Comment:
These two @param docblocks are incomplete (no variable name for product, no descriptions for both)